lib/components/project/user/row.vue

```diff
@@ -0,0 +1,119 @@
+<!--
+Copyright 2017 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/opendatakit/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <tr>
+    <td>{{ assignment.actor.displayName }}</td>
+    <td>
+      <form>
+        <div class="form-group">
+          <select ref="select" :value="selectedRole" :disabled="disabled"
+            :title="title" class="form-control" aria-label="Project Role"
+            @change="assignRole">
+            <option value="manager">Manager</option>
+            <option value="">None</option>
+          </select>
+          <span class="spinner-container">
+            <spinner :state="requesting"/>
```

Comment:
Should we not wrap a `<spinner>` in a `<span>`? (We do it in `FormEdit` as well.)